rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -233,6 +232,16 @@ public Blockchain getBlockchain() {
         return blockchain;
     }
 
+    public MiningMainchainView getMiningMainchainView() {
+        if (miningMainchainView == null) {
+            miningMainchainView = new MiningMainchainViewImpl(
+                    getBlockchain(),
+                    MiningConfig.REQUIRED_NUMBER_OF_BLOCKS_FOR_FORK_DETECTION_CALCULATION);
```

Comment:
I know this is a silly request before we have a linter in place, but could you stick to the current code style of this file? (the `);` bit goes in a new line).

Reply:
Sure, np!